File: app/src/net/artux/pda/Models/LoginUser.java
package net.artux.pda.Models;

public class LoginUser {

    private String emailOrLogin;
    private String password;

    public LoginUser(String emailOrLogin, String password) {
        this.emailOrLogin = emailOrLogin;
        this.password = password;
    }

    public String getEmailOrLogin() {
        return emailOrLogin;
    }

    public String getPassword() {
        return password;
    }
}

File: model/src/main/java/net/artux/pda/model/user/LoginUser.kt
package net.artux.pda.model.user;

import lombok.Data;

@Data
public class LoginUser {

    private final String emailOrLogin;
    private final String password;

    public LoginUser(String emailOrLogin, String password) {
        this.emailOrLogin = emailOrLogin;
        this.password = password;
    }

}
